source/validate_type_unique.cpp

```diff
@@ -36,7 +38,12 @@ spv_result_t TypeUniquePass(ValidationState_t& _,
     }
 
     if (!_.RegisterUniqueTypeDeclaration(*inst)) {
-      return _.diag(SPV_ERROR_INVALID_DATA)
+      // TODO(atgoo@github) Error logging temporarily disabled because it's
+      // failing vulkancts tests. Message in the diagnostics is for unit tests.
+      fprintf(stderr, "WARNING: Duplicate non-aggregate type declarations are"
```

Comment:
If I just return SPV_SUCCESS, then it won't be printed. Diagnostic is printed in tools/val/val.cpp, only if error code is not zero.

If I return SPV_WARNING, it won't be correctly handled (results are not checked for >= 0, but for == 0).